.github/workflows/build.yaml

```diff
@@ -143,3 +146,47 @@ jobs:
           APICULTURIST_GITHUB: ${{ secrets.APICULTURIST_GITHUB }}
           APICULTURIST_TOKEN: ${{ secrets.APICULTURIST_TOKEN }}
           monorepo: ${{ inputs.monorepo }}
+
+  dev_build:
+    runs-on: ubuntu-latest
+    needs: read_package
+    name: ${{ matrix.repos }}
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ${{fromJSON(needs.read_package.outputs.repoNames)}}
+    steps:
+      - name: Setup workspace
+        uses: qlik-oss/sn-gh-workflows/actions/setup@v3.0.1
+        with:
+          GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN || secrets.NPM_CI_TOKEN }}
+          NPMRC_ENCODED: ${{ secrets.QLIK_NPM_DEV_ENCODED }}
+      - name: Dev build
+        run: pnpm -r --filter ${{ matrix.repos }} build:dev
+      - name: Upload dev build
+        uses: qlik-trial/qmfe-workflows/actions/qmfe-to-aws@v1
+        with:
+          aws-access-key-id: ${{ secrets.QMFE_DEV_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.QMFE_DEV_ACCESS_KEY }}
+          qmfe-id: ${{ matrix.repos }}
+          source: ${{fromJSON(needs.read_package.outputs.repoFolders)[strategy.job-index]}}/dist
+          version: ${{ github.event.number }}
+          allow-overwrite: true
+          include-sourcemaps: true
+          aws-bucket-name: "hub-devbuilds"
+          s3-key: "nebula"
+      - name: Create comment
+        id: comment
+        uses: qlik-oss/sn-gh-workflows/actions/get-devbuild-comment@dvr/upload-devbuilds
+        with:
+          package: ${{ matrix.repos }}
+          namespace: "nebula"
+          version: ${{ github.event.number }}
+          file: ${{fromJSON(needs.read_package.outputs.systemJSFiles)[strategy.job-index]}}
+      - name: Comment the PR
+        if: github.event_name == 'pull_request'
+        uses: actions-ecosystem/action-create-comment@v1
```

Comment:
I used a different action in another project where it was possible to override the comment as well. That is probably better since you might have a lot of commits

Reply:
Or maybe we should only trigger when adding a comment like #devbuild 🤷

Reply:
I prefer to override a previous comment